rules/recommended.js

```diff
@@ -139,11 +139,20 @@ const baseRules = {
   'require-yield': 2,
   strict: [2, 'never'],
   'unicode-bom': [2, 'never'],
+  'unicorn/better-regex': 2,
+  'unicorn/custom-error-definition': 2,
   'unicorn/filename-case': [2, { case: 'kebabCase' }],
+  'unicorn/new-for-builtins': 2,
+  'unicorn/no-abusive-eslint-disable': 2,
   'unicorn/no-array-instanceof': 2,
   'unicorn/no-new-buffer': 2,
+  'unicorn/no-unreadable-array-destructuring': 2,
+  'unicorn/no-zero-fractions': 2,
+  'unicorn/prefer-includes': 2,
+  'unicorn/prefer-spread': 2,
```

Comment:
So, if creating an array from an Array-like object, we'd prefer .slice() over .from(), correct?

Reply:
This would enforce preferring spread, so `[...new Set([1,2,3])]`